src/main/java/org/opensearch/agent/tools/CreateAlertTool.java

```diff
@@ -275,7 +275,7 @@ public void init(Client client) {
         @Override
         public CreateAlertTool create(Map<String, Object> params) {
             String modelId = (String) params.get(MODEL_ID);
-            if (Strings.isBlank(modelId)) {
+            if (org.apache.commons.lang3.StringUtils.isBlank(modelId)) {
```

Comment:
Can you import the `StringUtils`?

Reply:
It's cherry-picked from PR #391, We better not change it in this backport PR. 

I think if we need to fix it, we better fix in in main branch and backport it to 2.x

Reply:
Oh I see. Is there any reason why you are directly using `org.apache.commons.lang3.StringUtils.isBlank` instead of importing it?